### PR TITLE
Adds download page

### PR DESCRIPTION
**Pull request by Soitora**
As per the Discord discussions.
Something small and easy, might be worked on more in the future but the goal is to get it indexed as the top result on Google.

image

### CODE REVIEW

**Comment by arkon** [comment]
Maybe you could also just modify the meta tags to improve the SEO?

**Comment by Soitora** [comment]
Meta tags for this page, or the main home page?

I think Frontmatter allows some meta data tags

**Comment by arkon** [comment]
The main home page.

**Comment by Soitora** [comment]
Aside from [adding meta tags through frontmatter](https://v1.vuepress.vuejs.org/guide/frontmatter.html#meta), SEO optimization has never been something I've been decent at. Not sure how to tweak it good enough for people to stop using other download sites

**Comment by GHOSTBEAR** [comment]
Something that I think could be added to this page is release notes for stable releases. But that could be addressed in another PR

**Comment by Soitora** [comment]
> Something that I think could be added to this page is release notes for stable releases. But that could be addressed in another PR

I am testing that right now, but got rate limited 😞

**Comment by Soitora** [comment]
Now has changelog (with correct markdown) too thanks to @Flat 



Anything else that should/could be tweaked?

File: src/download/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Download
+
+Download the latest version of **Tachiyomi** for your Android device.
+
+::: aside-guide
+Latest release: **<VersionTag/>**
+:::
+
+<DownloadButtons downloadStableTag="Tachiyomi" downloadPreviewTag="Tachiyomi Preview"/>
```
Comment (arkon) [inline src/download/README.md:15]: I don't remember, but does this distinguish what page the click metrics are being collected from?

Comment (Soitora) [reply src/download/README.md:15]: These in perticular is the name that gets recorded, so switched out for forks etc

It doesn't record which page

I'd use Google Tag Manager or something but it was too complex last I tried so silently gave up

**Comment by arkon** [comment]
I'd still consider just adding some tags to the main page too, like "download", "Android app", "official", "manga", etc.

**Pull request merged**